test/token-test.ts

```diff
@@ -0,0 +1,99 @@
+import { expect } from 'chai';
+import { Contract } from 'ethers';
+import {ethers} from 'hardhat';
+
+describe("LeoToken", function () {
+    let token: Contract;
+
+    const tokenCount = 100000;
+    const zeroAddress = ethers.constants.AddressZero;
+    const totalAmount = ethers.constants.WeiPerEther.mul( tokenCount ); // 1 wei has 18 decimals
+    const decimals = 18;
+
+    const messages = {
+        errorZeroAddress: 'Zero address is not allowed',
+        errorLackOfMoney: 'Not enough tokens',
+        errorNotAllowed: 'Not allowed to transfer tokens',
+        checkName: 'The name of token should be `Leocode Token`',
+        checkSymbol: 'The symbol of token should be `LEO`',
+        checkDecimals: 'The token should have 18 decimals',
+        checkTotalAmount: 'Check total amount of token',
+        checkSenderZeroAddress: 'Throw exception when sender address is null',
+        checkReceiverZeroAddress: 'Throw exception when receiver address is null',
+        checkTransactionAllowance: 'Throw exception when transaction doesn\'t have allowance',
+        checkNotEnoughTokens: 'Throw exception when sender doesn\'t have enough tokens',
+        checkBalanceAfterTransaction: 'Check account balance after transaction',
+        checkTransferEvent: 'Check Transfer Event',
+        checkAllowAndSent: 'Add to allowance, and transfer'
+    };
```

Comment:
To też fajne! Widzę to drugi raz u Was. Wcześniej nie spotkałem się z wyłuskaniem takich stringów przy testach do constów. Lubię to